Sources/Notifications/DatabaseConfiguration+Notifications.swift

```diff
@@ -0,0 +1,11 @@
+import Apodini
+
+extension DatabaseConfiguration {
+    /// Adds a database migration which is used by the `NotificationCenter`.
+    /// 
+    /// This will add the models: `DeviceDatabaseModel`, `DeviceTopic`, and `Topic` to the database.
+    public func addNotifications() -> Self {
+        _ = self.addMigrations(DeviceMigration())
```

Comment:
maybe we can add `@discarableResult` to `addMigrations(_:)` to avoid `_ =`.

Reply:
I agree with you. Just returning the result from `addMigrations(_:) ` also works